src/test/unit/test_tracer.py

```diff
@@ -376,25 +376,25 @@ def lambda_test_function(event, context):
 
     result = lambda_test_function({}, context)
     assert result == "ret_value"
-    assert SpansContainer.get_span().function_span[EXECUTION_TAGS_KEY] == [
-        {"key": key, "value": value}
-    ]
+    final_send = reporter_mock.call_args_list[-1][1]["msgs"]
+    enrichment_span = next(s for s in final_send if s["type"] == ENRICHMENT_TYPE)
```

Comment:
Can you extract it to a testing-util function and use this function in all the relevant places?